src/src/tx_main.cpp

```diff
@@ -428,7 +428,7 @@ bool ICACHE_RAM_ATTR ChannelIsClear()
   Radio.SetTxIdleMode();
   Radio.ClearIrqStatus(SX1280_IRQ_RADIO_ALL);
   Radio.SetDioIrqParams(SX1280_IRQ_RADIO_ALL, SX1280_IRQ_TX_DONE | SX1280_IRQ_RX_DONE, SX1280_IRQ_RADIO_NONE, SX1280_IRQ_RADIO_NONE);
-  return rssiResult < PowerEnumToLBTLimit((PowerLevels_e)config.GetPower());
+  return rssiResult < PowerEnumToLBTLimit((PowerLevels_e)config.currPower());
```

Comment:
This doesn't work. I need to be `POWERMGNT::currPower()` or maybe even better `config.GetDynamicPower() ? POWERMGNT::currPower() : (PowerLevels_e)config.GetPower()`

Reply:
Yeah it definitely needs to be the `POWERMGNT::currPower()` not the configured power.

Reply:
Fixed, I'm ashamed of the above commit, did it from my phone, never again, haha.